uavcan/node/430.GetInfo.0.1.uavcan

```diff
@@ -28,7 +28,7 @@ Version.1 software_version
 # For example, this field can be used for reporting the short git commit hash of the current
 # software revision.
 # Set to zero if not used.
-truncated uint64 software_vcs_revision_id
+uint64 software_vcs_revision_id
```

Comment:
why not truncated?

Reply:
Truncated makes sense only if narrowing conversions are expected. With 64-bit integer that would be redundant.